adapter/src/main/java/org/jvalue/ods/adapterservice/adapter/model/AdapterEvent.java

```diff
@@ -0,0 +1,94 @@
+package org.jvalue.ods.adapterservice.adapter.model;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.io.Serializable;
+
+/**
+ * This Event will be sent to the transformation service after data import execution.
```

Comment:
better say `will be published to the import queue`. The adapter should be decoupled from the consumer of the events it puts on the queue